ddp-workspace/projects/ddp-dsm-ui/src/app/phi-manifest/phi-manifest.model.ts

```diff
@@ -0,0 +1,41 @@
+export class PhiManifestModel {
+    constructor(public shortId: string, public firstName: string, public lastName: string,
+                public proxyFirstName: string, public proxyLastName: string, public dateOfBirth: string,
+                public gender: string, public dateOfPx: string, public facility: string, public sampleType: string,
+                public accessionNumber: string, public histology: string, public mfBarcode: string,
+                public normalCollaboratorSampleId: string, public blockId: string,
+                public tumorCollaboratorSampleId: string, public tissueSite: string, public sequencingResults: string,
+                public somaticAssentAddendumResponse: string, public somaticConsentTumorPediatricResponse: string,
+                public somaticConsentTumorResponse: string, public clinicalOrderDate: string,
+                public clinicalOrderId: string, public clinicalPdoNumber: string, public orderStatus: string,
+                public orderStatusDate: string) {
+    }
+
+    static parse(json): PhiManifestModel {
+        return new PhiManifestModel(json.shortId, json.firstName, json.lastName, json.proxyFirstName,
+            json.proxyLastName, json.dateOfBirth, json.gender, json.dateOfPx, json.facility, json.sampleType,
+            json.accessionNumber, json.histology, json.mfBarcode, json.normalCollaboratorSampleId, json.blockId,
+            json.tumorCollaboratorSampleId, json.tissueSite, json.sequencingResults, json.somaticAssentAddendumResponse,
+            json.somaticConsentTumorPediatricResponse, json.somaticConsentTumorResponse, json.clinicalOrderDate,
+            json.clinicalOrderId, json.clinicalPdoNumber, json.orderStatus, json.orderStatusDate);
+    }
+
+    public getReportAsMap(headers: string[], data: string[]): any {
+        const map: Map<string, string> = new Map();
+        for (let i = 0; i < headers.length; i++) {
+            const header = headers[i];
+            map[header] = this.convertValue(data[i]);
+        }
+        return map;
+    }
+
+    convertValue(value: string): string {
```

Comment:
If this is the only place that the report json is used, this conversion could be done server side to convert true/false into the string representations that the client is expecting.  PhiManifest or the service could do this, and it could be included in a server-side test.